doc/generate-kuadrant-auth-policy.md

```diff
@@ -422,40 +453,43 @@ The *reason* headers tell that `the API Key provided is invalid`.
 Using valid token (from the secret `cat-api-key-1` assigned to `POST /api/v1/cats`)
 in the `api_key` header should return 200 Ok
 
-```
-curl  -H "Host: example.com" -H "api_key: I_LIKE_CATS" -X POST -i "http://127.0.0.1:9080/api/v1/cat"
+```bash
+curl  -H "Host: example.com" -H "api_key: I_LIKE_CATS" -X POST -i "http://${INGRESS_IP}/api/v1/cat"
 ```
 
 - `GET /api/v1/dog` -> It's a protected endpoint with oidc (assigned to our keycloak instance and `petstore` realm)
 
 without credentials, it should return `401 Unauthorized`
 
 ```bash
-curl -H "Host: example.com" -i "http://127.0.0.1:9080/api/v1/dog"
+curl -H "Host: example.com" -i "http://${INGRESS_IP}/api/v1/dog"
 ```
 
+#### [Optional] SSO example
+
 To get the authentication token, this example is using Direct Access Grants oauth2 grant type
-(also known as Client Credentials grant type). When configuring the Keycloak (OIDC provider) client
+(also known as Resource Owner Password Credentials Grant grant type). When configuring the Keycloak (OIDC provider) client
 settings, we enabled Direct Access Grants to enable this procedure.
 We will be authenticating as `bob` user with `p` password.
 We previously created `bob` user in Keycloak in the `petstore` realm.
+We will use Command-line JSON processor `jq` to extract the access token into `ACCESS_TOKEN` variable:
 
-```
+```bash
 export ACCESS_TOKEN=$(curl -k -H "Content-Type: application/x-www-form-urlencoded" \
         -d 'grant_type=password' \
         -d 'client_id=petstore' \
         -d 'scope=openid' \
         -d 'username=bob' \
-        -d 'password=p' "https://${KEYCLOAK_PUBLIC_DOMAIN}/auth/realms/petstore/protocol/openid-connect/token" | jq -r '.access_token')
+        -d 'password=p' \
+        "${KEYCLOAK_TOKEN_ENDPOINT}" | jq -r '.access_token')
 ```
 
-> Replace `${KEYCLOAK_PUBLIC_DOMAIN}` with your SSO instance domain
-
+> Replace `${KEYCLOAK_TOKEN_ENDPOINT}` with your SSO instance token endpoint for your `petstore` realm.
```

Comment:
Same like the other var moving to be above where its needed so people can set it then run the bash command